pages/features/form-analytics.vue

```diff
@@ -0,0 +1,245 @@
+              item: 'https://formester.com/features/form-analytics/',
+            }
+          ],
+        },
+      ],
+    }
+  },
+  data() {
+    return {
+      features: [
+        {
+          title: 'Knowing Your Customers',
+          description:
+            'The data can be filtered by traffic source, device, location, browser, operating system, and other crucial factors. Using the advanced form analytics data, you can analyze user behavior and statistics over time. Adjust your form based on customer responses and optimize data collection.',
+          src: 'form-analytics/knowing-your-customers.svg',
+        },
+        {
+          title: 'Prevent form abandonment',
+          description:
+            'Forms having lengthy or irrelevant questions often lead to form abandonment. Advanced web form analytics help to identify specific drop-off points, enabling your business to optimize those areas and reduce abandonment rates by customers.',
+          src: 'form-analytics/track-your-traffic-by-analyzing-it.svg',
+        },
+        {
+          title: 'Track and analyze your traffic',
+          description:
+            'You can see in detail where your visitors are coming from, how many responded, what technology platforms they use, and which devices they are using to interact with your form in the traffic panel. It can help you prioritize the right questions.',
+          src: 'form-analytics/track-your-traffic-by-analyzing-it.svg',
+        },
+        {
+          title: 'Provide critical insight',
+          description:
+            'Conversion rate is crucial. When your conversion rate is low, you have a high view rate but little interaction. This indicates that your questions are too long or that there is room for improvement. Higher conversion rate means your online forms are getting the information as intended.',
+          src: 'form-analytics/provide-critical-insight.svg',
+        },
+      ],
+      threeStepContent: {
+        heading: "How to access robust form analytics in 3 simple steps:",
+        steps: [
+          {
+            title: "Sign up",
+            description: " Sign up for a free account on formester and build an online form or survey. You can also select any template from our pre-designed templates library."
+          },
+          {
+            title: "Form sharing",
+            description: "Share your form and wait for users to fill in the responses."
+          },
+          {
+            title: "Check the analytics",
+            description: "Go to the “Results” section to check the analytics and a detailed breakdown of all the form responses."
+          },
+        ]
+      }
+    }
+  },
+  async asyncData() {
+    let randomTestimonials = await allTestimonials
+    const randIndex = Math.floor(Math.random() * (randomTestimonials.length - 2))
+    randomTestimonials = randomTestimonials.slice(randIndex,  randIndex + 2);
+    return {randomTestimonials}
+  },
```

Comment:
we already have some utils for handling this. You can go through file 'formester-vs-google-form.vue' to see how it should be used.

Reply:
Okay @dhaladitya108 , i will go through that